auth/src/main/java/com/firebase/ui/auth/ui/email/CheckEmailFragment.java

```diff
@@ -132,6 +134,12 @@ public void onActivityCreated(@Nullable Bundle savedInstanceState) {
     }
 
     @Override
+    public void onSaveInstanceState(Bundle outState) {
+        outState.putBoolean(ExtraConstants.HAS_EXISTING_INSTANCE, true);
```

Comment:
Since use of the Bundle is not document behavior on FA, let's keep doing this your way.  FA could change that and not announce it.